### ISSUE TITLE
Crash on Furnace

### ISSUE THREAD

**Issue by FunnyBuddys** [iTXTech/Genisys#2309]
#### Issue description

Server crash while joining into a world with Furnace

#### Steps to reproduce the issue

Join to a world with Furnace
#### OS and versions

* Genisys: https://github.com/iTXTech/Genisys/commit/47e7f5d050e94e6c4c051c312df6ebd27b70143f
* PHP: 7.0
* Server OS: Debian 7
* Game version: 1.0.0.2

#### Crashdump, backtrace, memory dumps, plugins list or other files

```
[18:52:04] [Server thread/CRITICAL]: Error: "Call to a member function setItem() on null" (EXCEPTION) in "/src/pocketmine/tile/Furnace" at line 53
```

**Comment by JackNoordhuis** [comment]
Already fixed locally. I'm going to check out the other tiles to make sure nothing else was messed during all the merge conflicts before I push.

**Comment by FunnyBuddys** [comment]
okay!

**Comment by JackNoordhuis** [comment]
Fixed in [0838b68](https://github.com/iTXTech/Genisys/commit/0838b68354bdc302d1589eeff376a3f01a524665)
